addon/components/ember-thead/component.js

```diff
@@ -195,13 +195,6 @@ export default Component.extend({
   */
   containerWidthAdjustment: null,
 
-  /**
-    An action that is sent when sorts is updated
-    @argument onHeaderAction
-    @type Action?
-  */
-  onHeaderAction: null,
```

Comment:
This argument appears to be completely unused - I couldn't find reference to it anywhere else in the codebase (and the comment is obviously inaccurately copied from `onUpdateSorts` below).